go.mod

```diff
@@ -236,3 +236,5 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/onflow/flow-go v0.36.2-0.20240717214129-9ea6faeee3e7 => github.com/The-K-R-O-K/flow-go v0.35.14-crescendo-preview.27.0.20240718153829-58850e2c4673
```

Comment:
As mentioned in the other PRs, it would be great to avoid `replace` statements in `master`, as that effectively blocks all other PRs until it is removed again. It's error prone as we might accidentally tag a release with it and update downstream dependencies, where the replace statement is than _not_ in affect, leading to code with different versions

Reply:
Discussed this offline: Unfortunately, at the momement we do can't come up with a way to avoid replace statements for PRs from forks. So please go ahead, but remove the statement as soon as possible after the merge.